data/overview.yaml

```diff
@@ -40,30 +66,67 @@ group:
     convolution: ''
     examples for groups with a small cardinal: ''
 
-linear algebra:
-  vector space: 'algebra/module.html#vector_space'
-  linear map: 'algebra/module.html#linear_map'
-  product of vector spaces: 'algebra/pi_instances.html#prod.module'
-  vector subspace: 'algebra/module.html#subspace'
-  range of a linear map: 'linear_algebra/basic.html#linear_map.range'
-  kernel of a linear map: 'linear_algebra/basic.html#linear_map.ker'
-  quotient space: 'linear_algebra/basic.html#submodule.quotient'
-  sum of subspaces: 'linear_algebra/basic.html#submodule.complete_lattice'
-  direct sum: ''
-  complement subspace: ''
-  linear independent: 'linear_algebra/basis.html#linear_independent'
-  generating sets: ''
-  algebra of endomorphisms of a vector space: ''
-  general linear group: 'linear_algebra/basic.html#linear_map.general_linear_group'
-  invariant subspace of an endomorphism: ''
-  eigenvalue: ''
-  eigenvector: ''
-  linear representation: ''
-  irreducible representation: ''
-  finite dimension representations:
-    examples: ''
-    Schur's lemma: ''
+3. 
+Rings: 
+  Unitary rings:
+  ring morphisms: 
+  subrings: 
+  The ring Zn on the integers: 
+  Ideal of a commutative ring: 
+  quotient rings: 
+  first ideals: 
+  maximal ideals: 
+  Chinese remainder theorem: 
+  Commutative ring algebra:
+  Divisibility in commutative integral rings: 
+  Irreducible elements: 
+  invertible elements: 
+  first elements among them: 
+  Factorial rings: 
+  Greatest common divisor: 
+  least common multiple:
```

Comment:
I don't know if capital/lowercase matters, but just mentioning it's not really consistent
though don't really worry, if needed, I can just automatically rewrite them

Reply:
That would be great if you could automatically rewrite them in a new PR after this gets merged! I'm not committed to any capitalization scheme.